.ci/build.yaml

```diff
@@ -0,0 +1,33 @@
+parameters:
+  host: ''
+  pool: ''
+  sign: false
+
+jobs:
+- job: ${{ parameters.host }}
+  pool: ${{ parameters.pool }}
+  steps:
+  - ${{ if eq(parameters.host, 'Windows') }}:
+    - script: 'npm install -g esy@latest --unsafe-perm'
+      displayName: 'Install esy'
+  - ${{ if ne(parameters.host, 'Windows') }}:
+    - script: 'sudo npm install -g esy@latest --unsafe-perm'
+      displayName: 'Install esy'
+  - script: esy install
+    displayName: 'Running esy install and esy build'
+  - bash: |
+      PKG=~/foo/esy-ocaml
+      APP=~/foo/test-app
+      mkdir -p $PKG
+      mkdir -p $APP
+      cp -R * $PKG/
+      cp -R test-app/* $APP/
+  - ${{ if eq(parameters.sign, 'false') }}:
+      - bash: |
```

Comment:
```suggestion
      - script: |
```

Reply:
I think CI is failing because of these `bash` instead of `script` directives.
I've tried this before in esy-openssl and it worked. https://github.com/esy-packages/esy-openssl/pull/11/commits/de89c6581b4bf4877ed316842f053ba476172e7c